test/unit/spec/MfaVerify_spec.js

```diff
@@ -730,12 +730,40 @@ function (Okta,
             var button = test.form.submitButton();
             var buttonClass = button.attr('class');
             expect(buttonClass).toContain('link-button-disabled');
+            expect(button.prop('disabled')).toBe(true);
             return Expect.waitForFormError(test.form, test);
           })
           .then(function (test) {
             var button = test.form.submitButton();
             var buttonClass = button.attr('class');
+            expect($.ajax.calls.count()).toBe(1);
             expect(buttonClass).not.toContain('link-button-disabled');
+            expect(button.prop('disabled')).toBe(false);
+          });
+        });
+        itp('makes only one request if "verify button" was clicked twice', function () {
```

Comment:
but how are you clicking the ```submit``` button for the second time? It is disabled at that time. And the form has errors since the first time you clicked the submit button, right?